### ISSUE TITLE
Linux x86_64  : build broken

### ISSUE THREAD

**Issue by ebachard** [vitillo/power_gadget#2]
git clone 

Build is broken (see the log below) : 
````
somewhere_on_earth/Intel/power_gadget $ make clean && make
rm -f power_gadget librapl.so librapl.a msr.o cpuid.o rapl.o 
gcc -g -fpic -c msr.c cpuid.c rapl.c 
gcc -g -shared -o librapl.so msr.o cpuid.o rapl.o
gcc -g -c msr.c cpuid.c rapl.c 
ar rcs librapl.a msr.o cpuid.o rapl.o
gcc -g power_gadget.c -I. -o power_gadget -L. -lm  ./librapl.a
./librapl.a(rapl.o) : Dans la fonction « compute_Y » :
/home/eric/Devel/minidart/miniDart_TESTS_EN_COURS/Intel/power_gadget/rapl.c:519 : référence indéfinie vers « log »
collect2: error: ld returned 1 exit status
Makefile:14 : la recette pour la cible « power_gadget_static » a échouée
make: *** [power_gadget_static] Erreur 1
````
Solution : nothing to propose (probably some missing headers  ?)

Thanks !

-- 
ericb

**Comment by mlaves** [comment]
Hi @ebachard,
I had the same error when using the Makefile in Ubuntu 16.04. I fixed it by changing the linking order (see #3).

HTH, Max

Edit: I see that you already did a pull request 2 days ago (#1). This did not fix the issue for me. My pull request works for me™.

**Comment by ebachard** [comment]
@mlaves : you are right, it was indeed the linking order. I had a moment to study more deeply the issue and when I found log was the one in math.h (please don't laugh), I rapidly fixed it on my side. 
Thank you very much !

FYI, I commited the right change on my repo, but you probably did the same on your side :)

Closing the isue as fixed
